### ISSUE TITLE
Add ability to review changes as in github

### ISSUE THREAD

**Issue by iddm** [go-gitea/gitea#3580]
It would be very nice to have `review` feature or at least an ability to comment changes in the pull request.

**Comment by krombel** [comment]
@vityafx Might this be a duplicate of #124?

**Comment by iddm** [comment]
Nope. The `review` feature is a bit more complicated than just comments. Have you ever done a review on github? :) So no, this is not a duplicate. I'll fix the title so it reflects the intentions correctly. Thanks for the noticing this issue.

**Comment by lafriks** [comment]
Depends on #2583

**Comment by jonasfranz** [comment]
Closed by #3748

**Comment by iddm** [comment]
Good work, thanks!
